packages/frontend/src/connectors/frame/index.ts

```diff
@@ -100,15 +71,15 @@ export class FrameConnector extends Connector<JsonRpcProvider, unknown> {
         return chainId;
     }
 
-    public async getProvider({
-        chainId,
-        create,
-    }: { chainId?: number; create?: boolean } = {}) {
-        if (!this.provider || chainId || create) {
+    public async getProvider(config?: { chainId?: number }) {
+        if (!this.provider) {
```

Comment:
Should we add an `if(!chiandId) return` before this? Or it's not required?

Reply:
Nono by default it will connect to the currently selected chain in Frame. It's the intended behavior